src/modm/driver/color/tcs3472.hpp

```diff
@@ -205,10 +205,15 @@ class Tcs3472 : public tcs3472, public modm::I2cDevice<I2cMaster, 2>
 public:
 	Tcs3472(Data &data, uint8_t address = addr());
 
+	// Typical Runmodes
+	static constexpr Enable_t Enable_PollingMode = (Enable::POWER_ON | Enable::ADC_ENABLE);
+	static constexpr Enable_t Enable_InterruptMode = Enable_PollingMode | Enable::INTERRUPT_ENABLE;
+	static constexpr Enable_t Enable_InterruptMode_Waittime = Enable_InterruptMode | Enable::INTERRUPT_ENABLE;
```

Comment:
I've placed these predefined Runmodes bellow `tcs3472::Enable` first but gives error: called in a constant expression before its definition is complete.

Placing them here seems also being convinient.